operators/src/upgrade/config.rs

```diff
@@ -0,0 +1,90 @@
+use crate::upgrade::common::error::Error;
+use clap::ArgMatches;
+use core::time;
+use once_cell::sync::OnceCell;
+use openapi::tower::client::{ApiClient, Configuration};
+use url::Url;
+
+use super::{helm::client::HelmClient, k8s::client::K8sClient};
+
+static CONFIG: OnceCell<UpgradeOperatorConfig> = OnceCell::new();
+
+/// Upgrade operator config that can be passed through arguments.
+pub struct UpgradeOperatorConfig {
+    k8s_client: K8sClient,
+
+    helm_client: HelmClient,
+
+    mayastor_rest_client: ApiClient,
+
+    namespace: String,
+
+    chart_name: String,
```

Comment:
Is this the chart name or the release name? In helm terms release name is more appropriate, I think.